mmdet3d/models/roi_heads/bbox_heads/multi_group_head.py

```diff
@@ -0,0 +1,825 @@
+                -heatmap (torch.Tensor): Heatmap with the shape of \
+                    [B, N, H, W].
+        """
+        center_feat = self.feature_adapt_cls(x)
+        reg_feat = self.feature_adapt_reg(x)
+
+        cls_score = self.cls_head(center_feat)
+        ret = self.task_head(reg_feat)
+        ret['heatmap'] = cls_score
+
+        return ret
+
+
+@HEADS.register_module
+class CenterHead(nn.Module):
+    """CenterHead for CenterPoint.
+
+    Args:
+        mode (str): Mode of the head. Default: '3d'.
+        in_channels (list[int] | int): Channels of the input feature map.
+            Default: [128].
+        tasks (list[dict]): Task information including class number
+            and class names. Default: None.
+        dataset (str): Name of the dataset. Default: 'nuscenes'.
+        weight (float): Weight for location loss. Default: 0.25.
+        code_weights (list[int]): Code weights for location loss. Default: [].
+        common_heads (dict): Conv information for common heads.
+            Default: dict().
+        loss_cls (dict): Config of classification loss function.
+            Default: dict(type='GaussianFocalLoss', reduction='mean').
+        loss_bbox (dict): Config of regression loss function.
+            Default: dict(type='L1Loss', reduction='none').
+        seperate_head (dict): Config of seperate head. Default: dict(
+            type='SeparateHead', init_bias=-2.19, final_kernel=3)
+        share_conv_channel (int): Output channels for share_conv_layer.
+            Default: 64.
+        num_heatmap_convs (int): Number of conv layers for heatmap conv layer.
+            Default: 2.
+        conv_cfg (dict): Config of conv layer.
+            Default: dict(type='Conv2d')
+        norm_cfg (dict): Config of norm layer.
+            Default: dict(type='BN2d').
+        bias (str): Type of bias. Default: 'auto'.
+    """
+
+    def __init__(self,
+                 in_channels=[128],
+                 tasks=None,
+                 train_cfg=None,
+                 test_cfg=None,
+                 bbox_coder=None,
+                 common_heads=dict(),
+                 loss_cls=dict(type='GaussianFocalLoss', reduction='mean'),
+                 loss_bbox=dict(
+                     type='L1Loss', reduction='none', loss_weight=0.25),
+                 seperate_head=dict(
+                     type='SeparateHead', init_bias=-2.19, final_kernel=3),
+                 share_conv_channel=64,
+                 num_heatmap_convs=2,
+                 conv_cfg=dict(type='Conv2d'),
+                 norm_cfg=dict(type='BN2d'),
+                 bias='auto',
+                 norm_bbox=True):
+        super(CenterHead, self).__init__()
+
+        num_classes = [len(t['class_names']) for t in tasks]
+        self.class_names = [t['class_names'] for t in tasks]
+        self.train_cfg = train_cfg
+        self.test_cfg = test_cfg
+        self.encode_background_as_zeros = True
+        self.use_sigmoid_score = True
```

Comment:
This should be `self.use_sigmoid_cls`, and it's assignment should be the same as [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/dense_heads/anchor3d_head.py#L92). You can add assertion here to make sure users are using sigmoid.